integration-cli/docker_api_images_test.go

```diff
@@ -53,7 +55,7 @@ func (s *DockerSuite) TestAPIImagesSaveAndLoad(c *check.C) {
 	// TODO Windows to Windows CI: Investigate further why this test fails.
 	testRequires(c, Network)
 	testRequires(c, DaemonIsLinux)
-	buildImageSuccessfully(c, "saveandload", withDockerfile("FROM busybox\nENV FOO bar"))
+	buildImageSuccessfully(c, "saveandload", build.WithDockerfile("FROM busybox\nENV FOO bar"))
```

Comment:
Will these functions (`buildImage...`) be moved and become exported? It feels a bit odd that (e.g.) `build.WithDockerfile` is exported, but only consumed by an _unexported_ function 😅